Libraries/src/net/relinc/libraries/sample/TorsionSample.java

```diff
@@ -133,7 +132,7 @@ private static double getShearModulus(double youngsModulus, double poissonRatio)
 	
 	public double getShearModulus()
```

Comment:
getBarShearModulus at the very least. There may be a shear modulus of the sample that we care about at some point